app/javascript/components/public_health/CustomExport.js

```diff
@@ -214,46 +214,90 @@ class CustomExport extends React.Component {
                     checked={this.state.selected_records === 'current'}
                   />
                   {this.state.selected_records === 'current' && (
-                    <div className="px-1 pb-1">
-                      <Badge variant="secondary" className="mr-1">
-                        {`${_.capitalize(this.props.patient_query.workflow)} - ${this.props.tabs[this.props.patient_query.tab]?.label}`}
-                      </Badge>
+                    <div className="custom-export-filters">
+                      <p className="custom-export-filter-text">
+                        <small>
+                          <b className="mr-2">Linelist</b>
+                          {`${_.capitalize(this.props.patient_query.workflow)} - ${this.props.tabs[this.props.patient_query.tab]?.label}`}
+                        </small>
+                      </p>
                       {this.props.patient_query.jurisdiction !== this.props.jurisdiction.id && (
```

Comment:
this threw me off a little bit, because a jurisdiction is set but not shown.  should we just show regardless?

Reply:
resolving this based on Mike Nosal's feedback and your thumbs up to it: https://mitrecorp.slack.com/archives/C01A6P94R9B/p1626281752016900?thread_ts=1625589354.060100&cid=C01A6P94R9B